visualization/app/codeCharta/ui/screenshotButton/screenshotButton.component.html

```diff
@@ -1,3 +1,18 @@
-<md-button id="scrBtn" class="toolbar-button" ng-click="$ctrl.makeScreenshot()" title="ScreenShot ({{ $ctrl.SCREENSHOT_HOTKEY }})">
+<md-button
+	id="scrBtn"
+	class="toolbar-button"
+	ng-show="!$ctrl._viewModel.clipboardEnabled"
+	ng-click="$ctrl.makeScreenshotToFile()"
+	title="ScreenShot to file ({{ $ctrl.SCREENSHOT_HOTKEY_TO_FILE }})"
+>
+	<i class="fa fa-camera"></i>
+</md-button>
+<md-button
+	id="read"
```

Comment:
Are the id properties in use?
We could improve the name a bit (e.g. screenshotButtonClipboard, screenshotButtonFile)

Reply:
The Button is getting his style from a global css under the name toolbar-button.